source/ui.js

```diff
@@ -121,7 +121,12 @@ module.exports = async (options, pkg) => {
 			type: 'input',
 			name: 'tag',
 			message: 'Tag',
-			when: answers => options.publish && !pkg.private && version.isPrereleaseOrIncrement(answers.version) && !options.tag && !answers.tag,
+			when: answers =>
```

Comment:
Please avoid doing changes unrelated to the PR here.

Reply:
Oops, that was my editor. Should be fixed now.